recipes/hepmc2/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hepmc2" %}
+{% set version = "2.06.07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lcgapp.cern.ch/project/simu/HepMC/download/HepMC-{{ version }}.tar.gz
+  sha256: a0bdd6f36a3cc4cb59d6eb15cef9d46ce9b3739cae3324e81ebb2df6943e4594
+  folder: source
+  patches:
+    - patches/hepmc-2.03.06-reflex.patch
+    - patches/hepmc-2.06.07-WeightContainer-fix-size_type.patch
+    - patches/HepMC-2.06.07-nodoc.patch
+    - patches/HepMC-2.06.07-broken-tests.patch
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+
+  run:
+
+test:
+  source_files:
+    - source/
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+about:
+  home: http://hepmc.web.cern.ch/hepmc/
+  license: LGPL-2.1
+  license_family: LGPL
+  summary: 'HepMC - a C++ Event Record for Monte Carlo Generators'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+           The HepMC package is an object oriented, C++ event record
+           for High Energy Physics Monte Carlo generators and simulation.
+           The original code was written by M. Dobbs and J.B. Hansen
+            (Comput. Phys. Commun. 134 (2001) 41). Since the first 
+           implementation, the package has undergone several modifications
+           and in particular, the latest HepMC3 series is a completely new
+           re-write using currently available C++11 techniques. 
+
+extra:
+  recipe-maintainers:
+    - gartung
+    - chrisburr
+    - henryiii
```

Comment:
@henryiii Are you happy to be listed as a maintainer of this?

Reply:
Sure.